hooks/tk-multi-publish2/basic/submit_for_review.py

```diff
@@ -0,0 +1,300 @@
+﻿# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import nuke
+import os
+import sgtk
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class NukeSubmitForReviewPlugin(HookBaseClass):
+    """
+    Plugin for submitting a review from Nuke into Shotgun.
+
+    """
+
+    @property
+    def icon(self):
+        """
+        Path to an png icon on disk
+        """
+
+        # look for icon one level up from this hook's folder in "icons" folder
+        return os.path.join(
+            self.disk_location,
+            os.pardir,
+            "icons",
+            "review.png"
+        )
+
+    @property
+    def name(self):
+        """
+        One line display name describing the plugin
+        """
+        return "Submit for Review"
+
+    @property
+    def description(self):
+        """
+        Verbose, multi-line description of what the plugin does. This can
+        contain simple html for formatting.
+        """
+
+        loader_url = "https://support.shotgunsoftware.com/hc/en-us/articles/219033078"
+
+        return """
+
+        Submits a movie file to Shotgun for review. An entry will be
+        created in Shotgun which will include a reference to the movie file's current
+        path on disk. Other users will be able to access the file via
+        the review app on the Shotgun website.
+
+        <h3>File versioning</h3>
```

Comment:
All this stuff about versions can go away. It's really for the file versioning in the basic workflow.

Reply:
removed